--- a/configure.ac 2018-10-12 14:41:14.745090141 +0200
+++ b/configure.ac 2018-10-12 14:41:00.906009362 +0200
@@ -131,7 +131,7 @@
 AC_SUBST(DCMTK_LIBS)
 
 DCMTK_CFLAGS="-DHAVE_CONFIG_H -I\$(prefix)/include"
-DCMTK_LIBS="-L\$(prefix)/lib -ldcmjpeg -lijg8 -lijg12 -lijg16 -ldcmimage -ldcmimgle -ldcmnet -ldcmdata -loflog -lofstd"
+DCMTK_LIBS="-L\$(prefix)/lib -ldcmjpeg -lijg8 -lijg12 -lijg16 -ldcmimage -ldcmimgle -ldcmnet -ldcmdata -loflog -lofstd -licuuc"
 
 dnl set PREFIX
 if test "x${prefix}" = "xNONE"; then
